lib/Model/Message.php

```diff
@@ -52,6 +52,9 @@ class Message {
 	/** @var string */
 	protected $message = '';
 
+	/** @var string */
+	protected $info = '';
```

Comment:
Given how it is used and why it was introduced, maybe it would be better if called `$rawMessage` or `$unparsedMessage` or something like that instead?

Reply:
yeah, for the current usecases that would work. I though about using "extra" or "info" to be prepared for future stuff, like stickers and others (depending on how we do them).

Reply:
picked raw now